src/openforms/templates/includes/cookie-notice.html

```diff
@@ -4,7 +4,7 @@
 
 {% url 'cookie_consent_cookie_group_list' as manage_url %}
 
-{% if cookie_groups and request.path != manage_url %}
+{% if cookie_groups and request.path_info != manage_url %}
```

Comment:
shouldn't `manage_url` (which is just a `reverse` call) have applied the `SCRIPT_NAME` prefix already? So in that case, the `request.path` branch would be correct.

Reply:
`url` template tag doesn't use `request` in its implementation. Even when I test it with `SUBPATH=/of`, `manage_url` still equals `/cookies/`

Reply:
huh that's odd, because the django `url` tag or underlying `reverse` don't need this request object either. It relies on `get_script_prefix()` https://github.com/django/django/blob/40a2c811e3ed7fdb26ab4443e39e113c2fcf2aa9/django/urls/base.py#L109 which is a (thread) local.

```py
>>> from django.urls import get_script_prefix, set_script_prefix, reverse
>>> get_script_prefix()
'/'
>>> reverse("cookie_consent_cookie_group_list")
'/cookies/'
>>> set_script_prefix("/of")
>>> reverse("cookie_consent_cookie_group_list")
'/of/cookies/'

```